src/zimscraperlib/logging.py

```diff
@@ -59,25 +60,25 @@ def getLogger(  # noqa: N802
     if console:
         console_handler = logging.StreamHandler(console)
         console_handler.setFormatter(logging.Formatter(log_format))
-        console_handler.setLevel(level)  # pyright: ignore
+        console_handler.setLevel(level)
         logger.addHandler(console_handler)
 
     if file:
-        file_handler = RotatingFileHandler(  # pyright: ignore
+        file_handler = RotatingFileHandler(
             file,
-            maxBytes=file_max,  # pyright: ignore
-            backupCount=file_nb_backup,  # pyright: ignore
+            maxBytes=file_max,
+            backupCount=file_nb_backup,
         )
         file_handler.setFormatter(logging.Formatter(file_format or log_format))
-        file_handler.setLevel(file_level or level)  # pyright: ignore
+        file_handler.setLevel(file_level or level)
         logger.addHandler(file_handler)
 
     return logger
 
 
-def nicer_args_join(args: Iterable) -> str:
+def nicer_args_join(args: list[str]) -> str:
```

Comment:
I just fixed the type hint. Code is using `append` + it gets items by position, so it expects a `list`. I don't mind to extend this method if we have a use-case for tuple or something else, but this is a separate issue from my PoV (and it will not be breaking, so can be done later)